### PR TITLE
Fixes #1752 - Add milestone needstriage when create issue

### LINKED ISSUE #1752: Update /issues/new form endpoint to add needstriage milestone
We can automatically assign a milestone upon issue creation.
https://developer.github.com/v3/issues/#create-an-issue (see `milestone` param)

#1751 #886.

### PR DESCRIPTION
**Pull request by MDTsai**
Add milestone needstriage ID from app.config in post data when create issue. r?=@karlcow

### CODE REVIEW

File: webcompat/form.py [hunk 1]
```diff
@@ -187,15 +187,17 @@ def build_formdata(form_object):
     Here's a description of what the Issues API expects to create an issue
     (as of March 25, 2014):
```
Comment (karlcow) [inline webcompat/form.py]: nit: We probably want to change the date here. Or just remove the date entirely.

**Comment by MDTsai** [comment]
Remove the date from API description. r?=karlcow

**Comment by miketaylr** [comment]
I think this is the cause of #1779, you could fix that here and close out #1779 if you wanted.

**Comment by MDTsai** [comment]
r?=@karlcow fix test in #1779 now.

**Pull request merged**